### PR TITLE
Add Rules Page

### PR DESCRIPTION
**Pull request by rahulcs754**
Add rules page
image

### CODE REVIEW

File: pages/rule_page.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+              <li><a href="#"> About </a></li>
+              <li><a href="#"> Faq </a></li>
+            </ul>
+        </header> 
+
+    </div>
+</div>
+      
+    <!--- Header Part -->
+    <div class="flex flex-center flex flex-row">
```
Comment (tanishq20) [inline pages/rule_page.html]: used flex class two times

**Review by tanishq20 — COMMENTED** [review]
suggested small change.

**Review by shubhamkumar648 — APPROVED** [review]
as i seen consider the changes what Tanishq is commented , otherwise code is fine

File: pages/rule_page.html (hunk 1 above)
Comment (rahulcs754) [reply pages/rule_page.html]: remove two flex

**Pull request merged**